shotover/src/transforms/kafka/sink_cluster/mod.rs

```diff
@@ -1405,6 +1406,32 @@ impl KafkaSinkCluster {
         result
     }
 
+    /// This method removes all group ids from the DeleteGroups request and returns them split up by their destination.
```

Comment:
should change DeleteGroups to OffsetFetch?

Reply:
good catch, fixed